cloudflare/resource_cloudflare_logpush_job.go

```diff
@@ -0,0 +1,176 @@
+package cloudflare
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceCloudflareLogpushJob() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCloudflareLogpushJobCreate,
+		Read:   resourceCloudflareLogpushJobRead,
+		Update: resourceCloudflareLogpushJobUpdate,
+		Delete: resourceCloudflareLogpushJobDelete,
+
+		SchemaVersion: 0,
+		Schema: map[string]*schema.Schema{
+			"zone": {
```

Comment:
Could you change this to `zone_id`? We don't want zone names in resources any more. Plus skip the lookup below.

Reply:
Certainly - fixed in 3fb9208.